apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js

```diff
@@ -414,7 +414,7 @@ function computePointsFromRubric() {
       manualInput.value = manualPoints;
     }
   });
-  updatePointsView();
+  updatePointsView(this);
```

Comment:
I personally find the use of `this` to be hard to follow in code that isn't a member function of a `class`. I like the fact that arrow functions have dropped this-binding, making it harder to write code that uses this-binding via `call`/`apply`/`bind`.

Could we instead pass the `item`/`input` (hopefully with a better name) as an argument to `computePointsFromRubric()` (using an arrow function in the `addEventListener()` calls that set up the handlers)? Then we can explicitly pass this `item` down to `updatePointsView()`.

Reply:
Done in d9df5cbf2.